pkg/helm/init.go

```diff
@@ -175,14 +175,24 @@ func createCommonFiles(chartDir, chartName string, crd bool) error {
 			logrus.WithField("file", file).Info("created")
 		}
 	}
-	createFile(chartYAML(chartName), cDir, "Chart.yaml")
+	createFile(chartYAML(chartName, certManagerAsSubchart), cDir, "Chart.yaml")
 	createFile([]byte(helmIgnore), cDir, ".helmignore")
 	createFile(helpersYAML(chartName), cDir, "templates", "_helpers.tpl")
 	return err
 }
 
-func chartYAML(appName string) []byte {
-	return []byte(fmt.Sprintf(defaultChartfile, appName))
+func chartYAML(appName string, certManagerAsSubchart bool) []byte {
+	chartFile := defaultChartfile
+	annotatins := `
+dependencies:
+  - name: cert-manager
+    version: 1.9.1
```

Comment:
is it makes sense to remove subchart's `version` to always use the latest one?

Reply:
Yeah, make sense.